cloud/google/serviceaccount.go

```diff
@@ -65,6 +65,10 @@ func (gce *GCEClient) CreateMachineControllerServiceAccount(cluster *clusterv1.C
 		if err != nil {
 			return fmt.Errorf("couldn't grant permissions to service account: %v", err)
 		}
+		err = run("gcloud", "projects", "add-iam-policy-binding", project, "--member=serviceAccount:"+email, "--role=roles/iam.serviceAccountActor")
```

Comment:
I need a bit of context. Why does the machine controller service account need to impersonate other accounts?

Reply:
Once we got the master set up, there was an error message in the machine controller saying it needed access to the default service account when it was trying to create the node, so I added it here.

Reply:
I am also confused by this. Why does the machine controller service account need to be able to impersonate other service accounts? Shouldn't it just need the appropriate scopes to do what it needs and that's it? Or is this the permission that lets it create instances with service accounts?

Also, `iam.serviceAccountActor` has been deprecated. https://cloud.google.com/iam/docs/service-accounts#the_service_account_actor_role